index.bs

```diff
@@ -1564,6 +1565,90 @@ invoked, must run these steps:
 
 </div>
 
+# Drag and Drop Integration # {#drag-and-drop}
+
+During a <em>drag-and-drop operation</em>, dragged file and
+directory items are associated with [=file entries=] and [=directory entries=] 
+respectively. Each [=/entry=] is stored as an element in the <l spec=html>{{DragEvent|DragEvent's}}</l>
+<l spec=html>[=drag data store=] </l> and can be accessed using {{DataTransferItem/getAsFileSystemEntry()}} on
```

Comment:
nit: Just `<a spec=html>drag data store</a>`. The `<l>` bit can be useful for other link shorthands (i.e. `<l spec=html>{{Foo/bar()}}</l>` is shorthand for `<a method for=Foo spec=html>bar()</a>`), but there really isn't any benefit to it for regular "dfn" definitions.

Reply:
Done.